src/formatters/deployResultFormatter.ts

```diff
@@ -111,6 +99,12 @@ export class DeployResultFormatter implements Formatter<DeployResultJson> {
     this.displayReplacements();
   }
 
+  public determineVerbosity(): Verbosity {
+    if (this.flags.verbose) return 'verbose';
+    if (this.flags.concise) return 'concise';
+    return 'normal';
```

Comment:
override `TestResultFormatter.determineVerbosity()` because `project deploy <start | validate>` also support the `--concise` flag.